pkgs/applications/editors/vim/plugins/overrides.nix

```
@@ -3916,6 +3916,13 @@ in
     ];
   };
 
+  yaml-schema-detect-nvim = super.yaml-schema-detect-nvim.overrideAttrs {
+    dependencies = [
+      self.plenary-nvim
+      self.nvim-lspconfig
+    ];
```

A nit to maintain code consistency

```suggestion
    dependencies = with self; [
      plenary-nvim
      nvim-lspconfig
    ];
```

---

yep makes sense thank you :)